src/screens/Events/User/Register/EventRegister.jsx

```diff
@@ -306,23 +376,31 @@ const EventRegister = (event) => {
         <Col xs="12" lg="3">
           <Styled.VolunteerContainer>
             <Styled.VolunteerRow>
-              <Styled.SectionHeaderText>Fred Burger</Styled.SectionHeaderText>
+              <Styled.SectionHeaderText>{user.bio.first_name} {user.bio.last_name}</Styled.SectionHeaderText>
             </Styled.VolunteerRow>
             <Styled.VolunteerRow>
-              <Styled.DetailText>fredfredburger@gmail.com</Styled.DetailText>
-            </Styled.VolunteerRow>
-          </Styled.VolunteerContainer>
-        </Col>
-        <Col xs="12" lg="3">
-          <Styled.VolunteerContainer>
-            <Styled.VolunteerRow>
-              <Styled.SectionHeaderText>Carlos Burger</Styled.SectionHeaderText>
-            </Styled.VolunteerRow>
-            <Styled.VolunteerRow>
-              <Styled.DetailText>Minor</Styled.DetailText>
+              <Styled.DetailText>{user.bio.email}</Styled.DetailText>
             </Styled.VolunteerRow>
           </Styled.VolunteerContainer>
         </Col>
+        {events.minors && events.minors.map((minor) => (
+          <Row>
+            {minor.volunteer_id === user._id && minor.minor.map((names) => (
```

Comment:
This line should be above the `Row` tag. Other wise Row tag would be added even if the condition does not matches.